package.json

```
@@ -2150,7 +2150,7 @@
         "compile-webviews-verbose": "npx webpack --config webpack.datascience-ui.config.js",
         "postinstall": "node ./node_modules/vscode/bin/install && node ./build/ci/postInstall.js",
         "test": "node ./out/test/standardTest.js && node ./out/test/multiRootTest.js",
-        "test:unittests": "mocha --require source-map-support/register --opts ./build/.mocha.unittests.opts",
+        "test:unittests": "mocha --require source-map-support/register --opts ./build/.mocha.unittests.opts --grep='xUnit'",
```

Please remove the test grep...

```suggestion
        "test:unittests": "mocha --require source-map-support/register --opts ./build/.mocha.unittests.opts",
```